### PR TITLE
[FIX] Clarify that EDF/BDF files MUST have lower case extensions in BIDS

### PR DESCRIPTION
**Pull request by adam2392**
From https://github.com/bids-standard/bids-validator/pull/1381, we now clarify that the EDF files should be lower-case extension.

### CODE REVIEW

**Review by sappelhoff — APPROVED** [review]
Thanks Adam!

File: src/04-modality-specific-files/03-electroencephalography.md [hunk 1]
```diff
@@ -20,7 +20,10 @@ stored in one of the following formats:
 
 -   [European data format](https://www.edfplus.info/)
     (including [`edf+`](https://www.edfplus.info/specs/edfplus.html);
-    each recording consisting of a `.edf` file)
+    each recording consisting of a `.edf` file). Although the
+    European data format supports
+    both upper and lower case extensions, in BIDS the file extension MUST
+    be lower case (that is `.edf` instead of `.EDF`).
```
Comment (effigies) [inline src/04-modality-specific-files/03-electroencephalography.md:26]: I'm okay with this as-is, but I feel like it's over-explaining. We can simply say:

```suggestion
    each recording consisting of a `.edf` file).
    The capital `.EDF` extension MUST NOT be used.
```

Is it worth taking the opportunity to convert extensions to a table?

| Format | Extension(s) | Description |
|--|--|--|
| [European data format](https://www.edfplus.info/) | `.edf` | Each recording consists of a single file. [`edf+`](https://www.edfplus.info/specs/edfplus.html) files are permitted. The capital `.EDF` extension MUST NOT be used. |
| ... |

Comment (adam2392) [reply src/04-modality-specific-files/03-electroencephalography.md]: Converted to table.

File: src/04-modality-specific-files/03-electroencephalography.md [hunk 2]
```diff
@@ -18,19 +18,12 @@ The EEG community uses a variety of formats for storing raw data, and there is
...
+| [BrainVision Core Data Format](https://www.brainproducts.com/productdetails.php?id=21&tab=5) | `.vhdr`, `.vmrk`, `.eeg` | Each recording consisting of a  `.vhdr`, `.vmrk`, `.eeg` file triplet.                                                                                                                             |
+| [EEGLAB](https://sccn.ucsd.edu/eeglab)                                                       | `.set`, `.fdt`           | The format used by the MATLAB toolbox [EEGLAB](https://sccn.ucsd.edu/eeglab). Each recording consisting of a `.set` file with an optional `.fdt` file.                                             |
+| [Biosemi](https://www.biosemi.com/)                                                          | `bdf`                    | Each recording consists of a single `.bdf` file. [`bdf+`](https://www.teuniz.net/edfbrowser/bdfplus%20format%20description.html) files are permitted. The capital .BDF extension MUST NOT be used. |
```
Comment (hoechenberger) [inline src/04-modality-specific-files/03-electroencephalography.md:26]:
```suggestion
| [Biosemi](https://www.biosemi.com/)                                                          | `.bdf`                   | Each recording consists of a single `.bdf` file. [`bdf+`](https://www.teuniz.net/edfbrowser/bdfplus%20format%20description.html) files are permitted. The capital .BDF extension MUST NOT be used. |
```

File: src/04-modality-specific-files/04-intracranial-electroencephalography.md [hunk 3]
```diff
@@ -18,21 +18,13 @@ The iEEG community uses a variety of formats for storing raw data, and there is
...
+| [BrainVision Core Data Format](https://www.brainproducts.com/productdetails.php?id=21&tab=5) | `.vhdr`, `.vmrk`, `.eeg` | Each recording consisting of a  `.vhdr`, `.vmrk`, `.eeg` file triplet.                                                                                            |
+| [EEGLAB](https://sccn.ucsd.edu/eeglab)                                                       | `.set`, `.fdt`           | The format used by the MATLAB toolbox [EEGLAB](https://sccn.ucsd.edu/eeglab). Each recording consisting of a `.set` file with an optional `.fdt` file.            |
+| [Neurodata Without Borders](https://nwb-schema.readthedocs.io)                               | `nwb`                    | Each recording consists of a single `.nwb` file.                                                                                                                  |
+| [MEF3](https://msel.mayo.edu/files/codes/MEF%203%20Specification.pdf)                        | `mefd`                   | Each recording consisting of a `.mefd` directory.                                                                                                                 |
```
Comment (hoechenberger) [inline src/04-modality-specific-files/04-intracranial-electroencephalography.md:27]:
```suggestion
| [Neurodata Without Borders](https://nwb-schema.readthedocs.io)                               | `.nwb`                   | Each recording consists of a single `.nwb` file.                                                                                                                  |
| [MEF3](https://msel.mayo.edu/files/codes/MEF%203%20Specification.pdf)                        | `.mefd`                  | Each recording consisting of a `.mefd` directory.                                                                                                                 |
```

File: src/04-modality-specific-files/03-electroencephalography.md [hunk 4]
```diff
@@ -18,19 +18,12 @@ The EEG community uses a variety of formats for storing raw data, and there is
...
+| [BrainVision Core Data Format](https://www.brainproducts.com/productdetails.php?id=21&tab=5) | `.vhdr`, `.vmrk`, `.eeg` | Each recording consisting of a  `.vhdr`, `.vmrk`, `.eeg` file triplet.                                                                                                                             |
+| [EEGLAB](https://sccn.ucsd.edu/eeglab)                                                       | `.set`, `.fdt`           | The format used by the MATLAB toolbox [EEGLAB](https://sccn.ucsd.edu/eeglab). Each recording consisting of a `.set` file with an optional `.fdt` file.                                             |
+| [Biosemi](https://www.biosemi.com/)                                                          | `.bdf`                   | Each recording consists of a single `.bdf` file. [`bdf+`](https://www.teuniz.net/edfbrowser/bdfplus%20format%20description.html) files are permitted. The capital .BDF extension MUST NOT be used. |
```
Comment (sappelhoff) [inline src/04-modality-specific-files/03-electroencephalography.md:26]:
```suggestion
| **Format**                                                                                   | **Extension(s)**         | **Description**                                                                                                                                                                                      |
| -------------------------------------------------------------------------------------------- | ------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [European data format](https://www.edfplus.info/)                                            | `.edf`                   | Each recording consists of a single `.edf` file. [`edf+`](https://www.edfplus.info/specs/edfplus.html) files are permitted. The capital `.EDF` extension MUST NOT be used.                           |
| [BrainVision Core Data Format](https://www.brainproducts.com/productdetails.php?id=21&tab=5) | `.vhdr`, `.vmrk`, `.eeg` | Each recording consisting of a  `.vhdr`, `.vmrk`, `.eeg` file triplet.                                                                                                                               |
| [EEGLAB](https://sccn.ucsd.edu/eeglab)                                                       | `.set`, `.fdt`           | The format used by the MATLAB toolbox [EEGLAB](https://sccn.ucsd.edu/eeglab). Each recording consisting of a `.set` file with an optional `.fdt` file.                                               |
| [Biosemi](https://www.biosemi.com/)                                                          | `.bdf`                   | Each recording consists of a single `.bdf` file. [`bdf+`](https://www.teuniz.net/edfbrowser/bdfplus%20format%20description.html) files are permitted. The capital `.BDF` extension MUST NOT be used. |
```

File: src/04-modality-specific-files/04-intracranial-electroencephalography.md [hunk 5]
```diff
@@ -18,21 +18,13 @@ The iEEG community uses a variety of formats for storing raw data, and there is
...
+| [BrainVision Core Data Format](https://www.brainproducts.com/productdetails.php?id=21&tab=5) | `.vhdr`, `.vmrk`, `.eeg` | Each recording consisting of a  `.vhdr`, `.vmrk`, `.eeg` file triplet.                                                                                            |
+| [EEGLAB](https://sccn.ucsd.edu/eeglab)                                                       | `.set`, `.fdt`           | The format used by the MATLAB toolbox [EEGLAB](https://sccn.ucsd.edu/eeglab). Each recording consisting of a `.set` file with an optional `.fdt` file.            |
+| [Neurodata Without Borders](https://nwb-schema.readthedocs.io)                               | `.nwb`                   | Each recording consists of a single `.nwb` file.                                                                                                                  |
+| [MEF3](https://msel.mayo.edu/files/codes/MEF%203%20Specification.pdf)                        | `.mefd`                  | Each recording consisting of a `.mefd` directory.                                                                                                                 |
```
Comment (sappelhoff) [inline src/04-modality-specific-files/04-intracranial-electroencephalography.md:27]:
```suggestion
| **Format**                                                                                   | **Extension(s)**         | **Description**                                                                                                                                                            |
| -------------------------------------------------------------------------------------------- | ------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [European data format](https://www.edfplus.info/)                                            | `.edf`                   | Each recording consists of a `.edf` single file. [`edf+`](https://www.edfplus.info/specs/edfplus.html) files are permitted. The capital `.EDF` extension MUST NOT be used. |
| [BrainVision Core Data Format](https://www.brainproducts.com/productdetails.php?id=21&tab=5) | `.vhdr`, `.vmrk`, `.eeg` | Each recording consisting of a  `.vhdr`, `.vmrk`, `.eeg` file triplet.                                                                                                     |
| [EEGLAB](https://sccn.ucsd.edu/eeglab)                                                       | `.set`, `.fdt`           | The format used by the MATLAB toolbox [EEGLAB](https://sccn.ucsd.edu/eeglab). Each recording consisting of a `.set` file with an optional `.fdt` file.                     |
| [Neurodata Without Borders](https://nwb-schema.readthedocs.io)                               | `.nwb`                   | Each recording consists of a single `.nwb` file.                                                                                                                           |
| [MEF3](https://msel.mayo.edu/files/codes/MEF%203%20Specification.pdf)                        | `.mefd`                  | Each recording consisting of a `.mefd` directory.                                                                                                                          |
```

**Review by sappelhoff — APPROVED** [review]

File: src/04-modality-specific-files/03-electroencephalography.md [hunk 6]
```diff
@@ -18,19 +18,12 @@ The EEG community uses a variety of formats for storing raw data, and there is
...
+| [BrainVision Core Data Format](https://www.brainproducts.com/productdetails.php?id=21&tab=5) | `.vhdr`, `.vmrk`, `.eeg` | Each recording consisting of a  `.vhdr`, `.vmrk`, `.eeg` file triplet.                                                                                                                               |
+| [EEGLAB](https://sccn.ucsd.edu/eeglab)                                                       | `.set`, `.fdt`           | The format used by the MATLAB toolbox [EEGLAB](https://sccn.ucsd.edu/eeglab). Each recording consisting of a `.set` file with an optional `.fdt` file.                                               |
+| [Biosemi](https://www.biosemi.com/)                                                          | `.bdf`                   | Each recording consists of a single `.bdf` file. [`bdf+`](https://www.teuniz.net/edfbrowser/bdfplus%20format%20description.html) files are permitted. The capital `.BDF` extension MUST NOT be used. |
```
Comment (hoechenberger) [inline src/04-modality-specific-files/03-electroencephalography.md:26]: In the last column, in some rows we say `consists`, in others `consisting`.

Same in `04-intracranial-electroencephalography.md`

I think this should be made consistent

Comment (adam2392) [reply src/04-modality-specific-files/03-electroencephalography.md]: Done

**Pull request merged**